### PR TITLE
[texinfo/en] Learn Texinfo in Y minutes

### PR DESCRIPTION
**Pull request by roptat**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: texinfo.html.markdown [hunk 1]
```diff
@@ -0,0 +1,183 @@
+---
+language: Texinfo
+contributors:
+    - ["Julien Lepiller", "https://github.com/roptat"]
+filename: texinfo.texi
```
Comment (Menelion) [inline texinfo.html.markdown:5]:
```suggestion
filename: learntexinfo.texi
```

**Review by Menelion — APPROVED** [review]
Thanks for your contribution!

**Comment by roptat** [comment]
Hi, thanks for merging!

For some reason, I see the link on the main page, but it leads to a 404. `learntexinfo.texi` was correctly generated though, and is available for download. Did I do something wrong?

**Comment by roptat** [comment]
@Menelion sorry for the ping. It's been 10 days and although texinfo appears on the main page, it's still a 404. Do you know what's happening?

**Comment by Menelion** [comment]
I hate these display issues :( @adambard Sorry but could you please look into it? The filename seems correct.

**Comment by adambard** [comment]
I'll take a look when I'm at a computer, but my best guess is that the build is choking on the `texinfo` highlighting. This used to happen a lot more before we switched to pygments, but there are still languages that it doesn't recognize.

In this case the best option is to choose a neighboring language that does have support. As long as the alternative has the same comment syntax the results tend to be ok.

**Comment by roptat** [comment]
Indeed, there doesn't seem to be a texinfo lexer in pygments. They seem to just use the latex lexer, which is quite wrong, but I guess good enough.

**Pull request merged**